internal/service/advancedclustertpf/resource_schema.go

```diff
@@ -414,19 +399,19 @@ func SpecsSchema(markdownDescription string) schema.SingleNestedAttribute {
 	}
 }
 
-// TODO: generated from processArgs API endpoint
 func AdvancedConfigurationSchema(ctx context.Context) schema.SingleNestedAttribute {
 	return schema.SingleNestedAttribute{
-		Computed: true,
-		Optional: true,
-		// TODO: MaxItems: 1,
+		Computed:            true,
+		Optional:            true,
 		MarkdownDescription: "advanced_configuration", // TODO: add description
 		Attributes: map[string]schema.Attribute{
 			"change_stream_options_pre_and_post_images_expire_after_seconds": schema.Int64Attribute{
-				Optional: true,
-				Computed: true,
-				// Default:             int64default.StaticInt64(-1), // TODO: think if default in the server only
```

Comment:
@oarbusi I'm afraid I'm missing some context here. Can you confirm that it is safe to remove the `Default`?

Reply:
Discussed offline, added back in 7cd45086cd4ecf62538f0210d7b0a38715ce75bf